google/credential/tool.gpt

```diff
@@ -9,5 +9,7 @@ Share Credential: ../../oauth2 as google
         https://www.googleapis.com/auth/documents
         https://mail.google.com/
         https://www.googleapis.com/auth/gmail.readonly
-        https://www.googleapis.com/auth/gmail.compose" as scope
+        https://www.googleapis.com/auth/gmail.compose
+        https://www.googleapis.com/auth/drive
+        https://www.googleapis.com/auth/drive.file" as scope
```

Comment:
Why both `drive` and `drive.file`? `drive` is much more permissive.. so you wouldn't need both.

`drive.file` currently would not work very well with obot, because we don't have a workflow for changing the files you have given access to.